src/code/z_collision_check.c

```diff
@@ -2,11 +2,21 @@
 #include "terminal.h"
 #include "overlays/effects/ovl_Effect_Ss_HitMark/z_eff_ss_hitmark.h"
 
+typedef enum {
+    /* 0 */ MASSTYPE_IMMOVABLE,
+    /* 1 */ MASSTYPE_HEAVY,
+    /* 2 */ MASSTYPE_NORMAL
+} ColChkMassType;
```

Comment:
I do definitley prefer having this near its localized usage. Is this the only way to achieve the desired effect, or could you also use increment_block_number to achieve it?

Reply:
The block numbers work out perfectly without it which makes me think the enum definition was somewhere else in the original code (in a header maybe?). This adds 3 block numbers so we could move it back together with an `increment_block_number 253` to make it wrap all the way around, but I thought it was cleaner without that

Reply:
This feels, to me, to be the same caliber as trying to match something like line numbers in the code. Feel like we should prioritize keeping things organizationally in the file where we like them. but more opinions needed probably

Reply:
I don't feel strongly about it so I moved it back